### PR TITLE
Recipe view page

### PR DESCRIPTION
**Pull request by ljbarker**
Added a recipe view page so view button on recipe card works now

### CODE REVIEW

**Comment by adnunn19** [comment]
Also looks good!

**Review by eswenke — APPROVED** [review]

**Comment by bkennedy06** [comment]
Good stuff, I like the consistency between buttons. Super organized

File: packages/react-frontend/src/pages/Recipe.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { useParams } from 'react-router-dom';
```
Comment (bkennedy06) [inline packages/react-frontend/src/pages/Recipe.js:1]: this page looks good, I dig the organization

**Review by bkennedy06 — COMMENTED** [review]
Solid

**Comment by eswenke** [comment]
Good implementation, solid adaption from previous code.

**Pull request merged**